api/V1/recordingUtil.js

```diff
@@ -206,7 +216,8 @@ async function report(request) {
       audioBaitTime ? audioBaitTime.tz(config.timeZone).format("HH:mm:ss") : "",
       audioBaitDelta,
       audioBaitVolume,
-      urljoin(recording_url_base, r.id.toString())
+      urljoin(recording_url_base, r.id.toString()),
+      cacophonyIndex
```

Comment:
Is this going to be comma separated values in a CSV cell? That's a little gross.

How about separating the values with a semi-colon or forward slash?